### PR TITLE
Resolve #762, #738, #763, #764

### LINKED ISSUE #762: Make API invocations compatible with updated APIs

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/actions/walletActions.js [hunk 1]
```diff
@@ -343,23 +343,25 @@ function clearDecryptCloudWalletError () {
...
-    }
-  })
+  if (response) {
+    walletTypeList.forEach((walletType) => {
+      if (response[walletType]) {
+        rv[walletType] = { crypto: {} }
+        cryptoTypeList.forEach((cryptoType) => {
+          if (response[walletType].crypto && response[walletType].crypto[cryptoType]) {
+            rv[walletType].crypto[cryptoType] = [
+              response[walletType].crypto[cryptoType]
```
Comment (InVincible2016) [inline src/actions/walletActions.js]: this part needs to be updated, rv does not contain correct lastUesedWallet info

File: src/components/TransferFormComponent.jsx [hunk 2]
```diff
@@ -247,8 +247,8 @@ class TransferFormComponent extends Component<Props> {
               variant='outlined'
               error={!!formError.message}
```
Comment (InVincible2016) [inline src/components/TransferFormComponent.jsx]: !!formError.sendMessage

File: src/components/CancelReviewComponent.jsx [hunk 3]
```diff
@@ -133,25 +156,34 @@ class CancelReviewComponent extends Component {
...
         transferAmount,
         receiveTxHash,
         cancelTxHash,
         senderName,
         sender,
         destination,
+        receiverName,
         cryptoType,
-        cancelMessage
+        sendMessage
```
[the remark this answers is not in the document]
Comment (timothywangdev) [reply src/components/CancelReviewComponent.jsx]: this is actually right, it's on the review page where cancelMessage popup has not been shown

File: src/actions/walletActions.js (hunk 1 above)
Comment (timothywangdev) [reply src/actions/walletActions.js]: fixed

File: src/components/TransferFormComponent.jsx (hunk 2 above)
Comment (timothywangdev) [reply src/components/TransferFormComponent.jsx]: fixed

File: src/wallet.js [hunk 4]
```diff
@@ -30,7 +30,7 @@ if (['test', 'staging'].includes(env.REACT_APP_ENV)) {
 function getWalletStatus () {
   if (isMobile) {
     return {
-      disabled: true,
+      disabled: false,
```
[the remark this answers is not in the document]
Comment (timothywangdev) [reply src/wallet.js]: fixed

**Pull request merged**